src/main/java/com/google/codeu/servlets/ImageAnalysisServlet.java

```diff
@@ -100,17 +100,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                 
         if (flag == false)
             answer = "none";
-        imageHtml = imageHtml + "</ul><br/>"+ answer;
-
+        imageHtml = imageHtml + "</ul><br/>"+ answer; 
+        
         
         //save answer for use in level up
         HttpSession session = request.getSession();
         session.setAttribute("imageAnswer", answer);
 
+        Message imageUploaded = new Message(user, "Uploaded an image", level);
```

Comment:
do we want to send a message in the chat or do an alert of some kind?

Reply:
I found the alert method is hard to do for the same reason using one submit button is hard to do. Trying to call a servlet and javascript at the same time seems to not work well, only one of the calls goes through. This was the compromise I could think of without having to reorganize the code